september-util/src/main/java/edu/asu/diging/gilesecosystem/septemberutil/service/impl/SystemMessageHandler.java

```diff
@@ -50,6 +50,15 @@ public SystemMessageHandler(String applicationId) {
 
     @Override
     public void handleError(String msg, Exception exception) {
+        handleCommonException(msg, exception, ISystemMessageRequest.ERROR);
+    }
+
+    @Override
+    public void handleWarning(String msg, Exception exception) {
+        handleCommonException(msg, exception, ISystemMessageRequest.WARNING);
+    }
+
+    private void handleCommonException(String msg, Exception exception, String messageType) {
```

Comment:
@jdamerow changed handler method to take message type. If this is correct, I can make changes in Nepomuk and Giles for the same.